components/Icon.tsx

```diff
@@ -3,15 +3,27 @@ import clsx from "clsx";
 type Props = {
   type: "home" | "search";
   className?: string;
-  size?: string;
+  size?: "small" | "medium";
 };
 
 export default function Icon({ type, className, size }: Props) {
+  const sizes = {
```

Comment:
I'd define this outside of the function, that way it doesn't re-initialize every render